flags.go

```diff
@@ -84,6 +121,17 @@ func Enabled(ctx context.Context, name string) (enabled bool) {
 		stats.Gauge("goforit.flags.enabled", gauge, []string{fmt.Sprintf("flag:%s", name)}, .1)
 	}()
 
+	defer func() {
+		updateMtx.RLock()
+		staleness := time.Since(lastUpdate)
+		//histogram of cache process age
+		stats.Histogram("goforit.flags.process_cache_age_s", staleness.Seconds(), nil, .01)
+		if staleness > STALENESS_THRESHOLD && time.Since(lastAssert) > 60*time.Second {
+			lastAssert = time.Now()
+			//TODO send to sentry
```

Comment:
For now, let's just log a message here

Reply:
Added some logging throughout the file, but just does a Printf for now